cmd/gonews/main.go

```diff
@@ -15,9 +19,10 @@ import (
 const botToken = "BOT_TOKEN"
 
 var (
-	lifeTime   = kingpin.Flag("lifetime", "Life time of posts").Default("24h").Duration()
-	updateTime = kingpin.Flag("updtime", "Posts update time").Default("1h").Duration()
-	scrapeURL  = kingpin.Flag("url", "Scrape rss channel").Default("https://habr.com/ru/rss/hubs/all/").String()
+	lifeTime       = kingpin.Flag("lifetime", "Life time of posts").Default("24h").Duration()
+	updateTime     = kingpin.Flag("updtime", "Posts update time").Default("1h").Duration()
+	scrapeURL      = kingpin.Flag("url", "Scrape rss channel").Default("https://habr.com/ru/rss/hubs/all/").String()
+	configFilePath = kingpin.Flag("config", "Path to config file of synonyms").Default("synonyms-config.yml").String()
```

Comment:
IMO better refer to it as a config file, not synonyms file, in case you will add more tuning options